presto-hive/src/main/java/com/facebook/presto/hive/metastore/thrift/ThriftMetastoreModule.java

```diff
@@ -47,6 +47,8 @@ protected void setup(Binder binder)
 
         binder.bind(HiveMetastore.class).to(ThriftHiveMetastore.class).in(Scopes.SINGLETON);
 
+        configBinder(binder).bindConfig(ThriftHiveMetastoreConfig.class);
```

Comment:
add this just before `binder.bind(HiveMetastore.class).to(ThriftHiveMetastore.class...` line